Adyen/Core/APIClient/APIEnvironment.swift

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// :nodoc:
+/// Represents any API and its different environments (test, beta, and live).
+public protocol APIEnvironment {
```

Comment:
This protocol is never used..

Reply:
sorry, this file slipped through the cracks of the merge conflicts, removed